docs/development.rst

```diff
@@ -388,14 +388,19 @@ recommended process for generating a new release is described here.
 
         python setup.py sdist upload
 
+.. _releasepythonclientpackage:
+
 Releasing the python client package
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Whenever the main Girder package is released, the python client package should also
-be versioned and released if is has changed since the last Girder release or the last
-time it was released. Normal semantic versioning is not in use for the python client
-package because its version is partially dependent on the Girder server package
-version. The rules for versioning the python client package are as follows:
+The design intent behind the python client package is to work with as many
+versions of the Girder server as possible; think carefully before breaking this
+compatibility. Whenever the main Girder package is released, the python client
+package should also be versioned and released if is has changed since the last
```

Comment:
This was actually a reference to the famous quote `It depends upon what the meaning of the word 'is' is`, so when 'is' has changed...

Naw, good catch.